api/Event.json

```
@@ -59,7 +59,7 @@
               "version_added": "18"
             },
             "edge": {
-              "version_added": "≤18"
+              "version_added": "≤13"
```

Looks like it was supported [from the first release](https://github.com/MicrosoftEdge/Status/pull/190)

```suggestion
              "version_added": "12"
```